app/src/androidTest/java/com/jens/automation2/tests/CreateSimpleProfileTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.jens.automation2.tests
+
+import androidx.test.ext.junit.rules.activityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import com.jens.automation2.ActivityMainTabLayout
+import com.jens.automation2.screenobjects.CreateProfileScreen
+import com.jens.automation2.screenobjects.ProfilesScreen
+import com.jens.automation2.screenobjects.Strings.simpleProfileName
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class CreateSimpleProfileTest {
+
```

Comment:
Please make style consistent in all classes - if you want this blank line please add it to each class. if no - then remove it from each class